README.rst

```diff
@@ -37,56 +37,45 @@ Usage Example
 .. code:: python
 
     import time
+    import random
     import board
     import terminalio
     from adafruit_matrixportal.matrixportal import MatrixPortal
 
-    # You can display in 'GBP', 'EUR' or 'USD'
-    CURRENCY = "USD"
-    # Set up where we'll be fetching data from
-    DATA_SOURCE = "https://api.coindesk.com/v1/bpi/currentprice.json"
-    DATA_LOCATION = ["bpi", CURRENCY, "rate_float"]
+    # --- Display setup ---
+    matrixportal = MatrixPortal(status_neopixel=board.NEOPIXEL, debug=True)
 
-
-    def text_transform(val):
-        if CURRENCY == "USD":
-            return "$%d" % val
-        if CURRENCY == "EUR":
-            return "‎€%d" % val
-        if CURRENCY == "GBP":
-            return "£%d" % val
-        return "%d" % val
-
-
-    # the current working directory (where this file is)
-    cwd = ("/" + __file__).rsplit("/", 1)[0]
-
-    matrixportal = MatrixPortal(
-        url=DATA_SOURCE,
-        json_path=DATA_LOCATION,
-        status_neopixel=board.NEOPIXEL,
-        debug=True,
+    # Create a new label with the color and text selected
+    matrixportal.add_text(
+        text_font=terminalio.FONT,
+        text_position=(0, (matrixportal.graphics.display.height // 2) - 1),
+        scrolling=True,
     )
 
+    # Static 'Connecting' Text
     matrixportal.add_text(
         text_font=terminalio.FONT,
-        text_position=(16, 16),
-        text_color=0xFFFFFF,
-        text_transform=text_transform,
+        text_position=(2, (matrixportal.graphics.display.height // 2) - 1),
     )
-    matrixportal.preload_font(b"$012345789")  # preload numbers
-    matrixportal.preload_font((0x00A3, 0x20AC))  # preload gbp/euro symbol
 
-    while True:
-        try:
-            value = matrixportal.fetch()
-            print("Response is", value)
-        except (ValueError, RuntimeError) as e:
-            print("Some error occured, retrying! -", e)
+    SCROLL_DELAY = 0.03
 
-        time.sleep(3 * 60)  # wait 3 minutes
+    contents = [
+        { 'text': 'THIS IS RED',  'color': '#cf2727'},
+        { 'text': 'THIS IS BLUE', 'color': '#0846e4'},
+    ]
+
+    matrixportal.set_text(" ", 1)
+    while True:
+        for content in contents:
+            matrixportal.set_text(content['text'])
 
+            # Set the text color
+            matrixportal.set_text_color(content['color'])
 
+            # Scroll it
+            matrixportal.scroll_text(SCROLL_DELAY)
+        
```

Comment:
I think maybe it wants an extra newline here at the end of the code snippet. 

Thanks for working on this!